### PR TITLE
add statesync to root.go

### PR DESCRIPTION
**Pull request by faddat**
## Description and Motivation

This PR will fix state sync for migaloo v1.0.x.  Migaloo v1.0.x was missing the configuration needed for 
successful state sync in root.go.

## Related Issues

---
## Checklist:

- [ ] I have read [Migaloo's contribution guidelines](https://github.com/White-Whale-Defi-Platform/migaloo-chain/blob/main/docs/CONTRIBUTING.md).
- [ ] My pull request has a sound title and description (not something vague like `Update index.md`)
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation.
- [ ] The code is formatted properly `golangci-lint run ./... --fix`.

### CODE REVIEW

File: scripts/statesync.bash [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+
+# Initialize chain.
+migalood init test 
+
+# get genesis
+wget -O ~/.migalood/config/genesis.json https://github.com/White-Whale-Defi-Platform/migaloo-chain/raw/release/v2.0.x/networks/mainnet/genesis.json
+
+# Set minimum gas price.
+sed -i'' 's/minimum-gas-prices = ""/minimum-gas-prices = "0.0025uwhale"/' $HOME/.migalood/config/app.toml
```
[the remark this answers is not in the document]
Comment (hoangdv2429) [reply scripts/statesync.bash:27]: add one additional space so that this can also work on mac.

**Comment by faddat** [comment]
@hoangdv2429 some comments, I've fixed the issues:

* a script that users run shouldn't remove files from their filesystem
* when debugging this script from your laptop, you probably want to invoke it like:

```bash
bash scripts/statesync.bash
```

* everything looks good now, can you please:
  * make another PR that removes the notional private endpoint from this script
  * re-verify in that PR
  * make a github action that tests state sync so that we always know that it works

**Comment by hoangdv2429** [comment]
I'll do it now

**Comment by faddat** [comment]
Excellent thank you

**Pull request merged**